themes/mt.omp.json

```diff
@@ -16,50 +16,128 @@
           "background": "#FFFF00",
           "foreground": "#000000",
           "powerline_symbol": "\ue0b0",
+          
           "properties": {
             "style": "folder"
           },
           "style": "powerline",
-          "template": " {{ .Path }} ",
+          "template": " <#5800B3>\uf07b</>  {{ .Path }} ",
           "type": "path"
         },
         {
           "background": "#4E44FF",
           "foreground": "#ffffff",
           "powerline_symbol": "\ue0b0",
+          "leading_diamond": "\ue0b6",
+          "trailing_diamond": "<#28bd03>\ue0b4</> ",
           "properties": {
             "branch_icon": "",
             "fetch_stash_count": true,
             "fetch_status": false,
             "fetch_upstream_icon": true
           },
           "style": "powerline",
-          "template": " \u279c ({{ .UpstreamIcon }}{{ .HEAD }}{{ if gt .StashCount 0 }} \ueb4b {{ .StashCount }}{{ end }}) ",
+          "template": " \ue725 \u279c ({{ .HEAD }}{{ if gt .StashCount 0 }} \ueb4b {{ .StashCount }}{{ end }}) ",
           "type": "git"
+        }
+      ],
+      "type": "prompt"
+    },
+    {
+      "alignment": "right",
+      "segments": [
+        {
+          "foreground": "#a9ffb4",
+          "style": "plain",
+          "type": "text"
+        },
+        {
+          "background": "#FF8000",
+          "foreground": "#ffffff",
+          "leading_diamond": "\ue0b6",
+          "style": "diamond",
+          "template": "<#ffffff> 🦀</> {{ if .Error }}oops{{ else }}{{ .Full }}{{ end }} ",
+          "trailing_diamond": "<#FF8000>\ue0b4</> ",
+          "type": "rust"
         },
         {
-          "background": "#4e903d",
+          "background": "#28bd03",
           "foreground": "#ffffff",
+          "leading_diamond": "\ue0b6",
           "powerline_symbol": "\ue0b0",
-          "style": "powerline",
+          "style": "diamond",
           "template": " \ue718 {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }} ",
+          "trailing_diamond": "<#28bd03>\ue0b4</> ",
           "type": "node"
         },
         {
-          "background": "#16C60C",
-          "foreground": "#ffffff",
+          "foreground": "#a9ffb4",
           "properties": {
-            "time_format": "15:04"
+            "style": "dallas",
+            "threshold": 0
           },
           "style": "diamond",
+          "template": "\uf120 {{ .FormattedMs }}s <#ffffff></>",
+          "type": "executiontime"
+        },
+        {
+          "properties": {
+            "root_icon": "\uf292 "
+          },
+          "style": "diamond",
+          "template": " \uf0e7 ",
+          "type": "root"
+        },
+        {
+          "background": "#FFFF00",
+          "foreground": "#000000",
+          "leading_diamond": "\ue0b6",
+          "style": "diamond",
+          "template": " <#000000>🍂 (RAMus)</> {{ round .PhysicalPercentUsed .Precision }}/1e2 ",
```

Comment:
I would remove this as the below indicates the same but without percentages.

Reply:
😺🍁 I decided this part is a bit unnecessary, what do u think of that? :

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/69170710/9e73c89d-dd3e-4f5b-b8ec-23b779209f99)